src/components/PhotoListForm/PhotoListForm.tsx

```diff
@@ -43,32 +86,65 @@ const PhotoListForm: React.FC<WeddingData> = ({guests, changeView}) => {
   }
 
   const submitPhoto = (event: React.FormEvent) => {
+    setIsLoading(true)
     event.preventDefault()
-    const guestList = guestsOptions.filter((guest: any) => guest.isChecked)
-    if (guestList.length >= 1) {
+    const checkedGuests = guestsOptions.filter((guest: any) => guest.isChecked)
+    if(checkedGuests.length >= 1) {
       setIsError(false)
       setErrorMessage('')
       const newPhotoData = {
         id: photoData.length + 1,
         photoNumber: photoData.length + 1,
-        guests: guestList,
+        guests: checkedGuests,
         description: description
       }
-      setPhotoData([...photoData, newPhotoData])
+      const guestIds = newPhotoData.guests.map((guest:any) => guest.id)
+      const postPhoto = {
+        number: newPhotoData.photoNumber,
+        description: newPhotoData.description,
+        guest: guestIds,
+        weddingId: weddingId
+      }
+      setPhotoData([...photoData, postPhoto])
       setDescription('')
+      postAPhoto(postPhoto)
     } else {
       setIsError(true)
       setErrorMessage('Please select at least one guest for the photo')
     }
-
+    setIsLoading(false)
     // conditional rendering - make sure guestlist,length is at least 1?
```

Comment:
Is this note for future or just a 'for your information'?

Reply:
I wasn't sure. I think @hannakim91 left that note and I didn't want to delete it in case it was important

Reply:
I remember getting rid of this comment at some point (it's happening in the `else` statement 111-114 but it returns! I'll make a ticket of it to clean up unneeded comments